src/main/java/com/danielagapov/spawn/Services/OAuth/OAuthService.java

```diff
@@ -41,6 +45,17 @@ public UserDTO makeUser(UserDTO userDTO, String externId) {
         return userDTO;
     }
 
+    @Override
+    public Map<Boolean, FullUserDTO> isUser(String externId) {
```

Comment:
```suggestion
    public FullUserDTO getUserIfExistsByExternalId(String externalUserId) {
```
If you go with your implementation, I think this method signature is more fitting

Reply:
Addressed in [here](2959537cd923411da8e8f36ad29e2fc5a3601624)